MODULE.bazel

```diff
@@ -43,7 +43,7 @@ bazel_dep(
 )
 bazel_dep(
     name = "rules_kotlin",
-    version = "2.0.0",
+    version = "2.1.3",
```

Comment:
this causes CI to fail on Windows

Reply:
Then we might have a problem, because apparently kotlin plugin 252 was compiled with kotlin 2.2.0 version and required newer version of rules than 2.0.0